src/rules/color-no-invalid-hex/__tests__/index.js

```diff
@@ -0,0 +1,18 @@
+import { ruleTester } from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule(null, tr => {
+  tr.ok("a { color: pink; }")
+  tr.ok("a { color: #000; }")
+  tr.ok("a { something: #000, #fff, #ababab; }")
```

Comment:
Yep.

Color is cool for color manipulation and we are not doing any manipulation so maybe too much for now. That said, we might need it later so I don't really care, I was just thinking it can be pretty faster to handle with a "simple" regex (` [0-9a-fA-F]{3,4}|[0-9a-fA-F]{6}|[0-9a-fA-F]{8}`).

FYI I am a owner for this color module (but haven't done anything with it yet), so if you want we can move forward with this module easily.